packages/ui/src/components/MaterialsCard/MaterialsCard.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Box, BoxProps } from '@chakra-ui/react';
+
+import { Button } from '../Button';
+import { Typography } from '../Typography/Typography';
+
+type MaterialsCardStatus = 'idle' | 'loading' | 'generated';
```

Comment:
💭 I think better way is used ,,const enum'' because we used this value in a few spots (MaterialsCard, MateriaslCard.test, MaterialCard.stories) so if we'll want to change somtehing in future it will be easier. 
Second reason  is we don't have to  create again status object in MateriaCard.stories.tsx we'll just assign const enum. (line 7)

const MaterialsCardStatus = {
  idle: 'idle',
  loading: 'loading',
  generated: 'generated',
} as const;

Reply:
I'm not sure if it's beneficial as in such a case we would need to always have two imports (component and enum). The question is, how it makes future changes easier? If we change the name of one value we will need to replace it in every place it's used anyway.

Reply:
yes, but in my opinion, it is better to change the variable than a string and with this convention never miss a place that needs to be updated

Reply:
It seems like a really good argument when you create a JavaScript app where misspelling some string or forgetting to update it somewhere are quite common problems. I think when it comes to TypeScript those problems cannot happen as you will be notified in the console (and probably in your IDE as well).